python_modules/libraries/dagster-k8s/dagster_k8s/client.py

```diff
@@ -749,3 +749,10 @@ def get_pod_debug_info(self, pod_name, namespace, container_name: Optional[str]
             + (f"\n\n{log_str}" if log_str else "")
             + f"\n\n{warning_str}"
         )
+
+    def create_namespaced_job_with_retries(self, body: V1Job, namespace: str):
+        k8s_api_retry(
+            lambda: self.batch_api.create_namespaced_job(body=body, namespace=namespace),
+            max_retries=3,
+            timeout=15.0,
```

Comment:
it'd be nice to make this configurable, I wonder where that should live though

Reply:
how about as arguments to this function with defaults set?

Reply:
15 seconds pretty high for an initial retry to me - should we get fancy and throw some exponential backoff in here?

Reply:
reduced to 10 seconds. I miscounted- the default that we use for all the other apis (4 attempts, 10 seconds pause) will get us the 30 seconds we need here

exponential backoff sounds good, I don't think it will make much difference in this case though